src/main/java/iroha/validation/transactions/signatory/impl/TransactionSignerImpl.java

```diff
@@ -15,6 +15,10 @@
 
   private final IrohaAPI irohaAPI;
   private final KeyPair keyPair;
+  private final KeyPair fakeKeyPair = Utils.parseHexKeypair(
+      "0000000000000000000000000000000000000000000000000000000000000000",
```

Comment:
Why?

Reply:
It's just a way to fail transaction in Iroha if it fails validation in BRVS

Reply:
Can we make it static?